xclim/indices/_threshold.py

```diff
@@ -2107,6 +2108,56 @@ def tropical_nights(
     return tn_days_above(tasmin, thresh=thresh, freq=freq)
 
 
+@declare_units(pr="[precipitation]", prc="[precipitation]", thresh="[precipitation]")
+def rprctot(
+    pr: xarray.DataArray,
+    prc: xarray.DataArray,
+    thresh: str = "1.0 mm/day",
+    freq: str = "YS",
+) -> xarray.DataArray:
+    """Proportion of accumulated precipitation arising from convective processes.
+
+    Return the proportion of total accumulated precipitation due to convection on days with total precipitation exceeding a specified threshold during the given period.
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
+      Daily precipitation.
+    prc : xarray.DataArray
+      Daily convective precipitation.
+    thresh : str
+      Precipitation value over which a day is considered wet.
+    freq : str
+      Resampling frequency.
+
+    Returns
+    -------
+    xarray.DataArray, [time]
+      The proportion of the total precipition accounted for by convective precipitation for each period [1].
+
+    Examples
+    --------
+    The following would compute for each grid cell of file `pr.day.nc` the proportion of total rainfall
+    arising from convective precipitation on days with more than 5mm rainfall at the seasonal frequency, ie DJF, MAM, JJA, SON, DJF, etc.:
+
+    >>> from xclim.indices import rprctot
+    >>> pr = xr.open_dataset(path_to_pr_file).pr
+    >>> prc = xr.open_dataset(path_to_prc_file).prc
```

Comment:
This may be a bit strange, but `path_to_pr_file` is an existing file that we link to by renaming it in the doctest namespace (https://github.com/Ouranosinc/xclim/blob/118441f89d221cfffbd2e1fd0b966517e731378d/xclim/testing/tests/conftest.py#L391).

My suggestion would the following:

1. add a minimal working example netcdf to the xclim testing data (https://github.com/Ouranosinc/xclim-testdata) that has both `pr` and `prc`,
2. add a link to it in `xclim/testing/conftest.py` using the name `path_to_prc_file`,
3. in this doctest, ensure that both `pr` and `prc` are being sourced from the same file.

Reply:
There's not an obvious place to put the data in xclim-testdata. Any suggestions?

Reply:
Also, are you wanting a minimal example with real data or are random values fine?

Reply:
Depending on _what_ the data is, I would place it in a corresponding folder, ie: if you gather a few lats/lons from ERA5, I would place them in here (https://github.com/Ouranosinc/xclim-testdata/tree/main/ERA5). It doesn't have to be real values either. Just a minimal working example is all we ask.

Reply:
Does `prc` exist in ERA5? The easiest would be to add it to "ERA5/daily_surface_cancities.nc".

If not :
If its random data, it's easier to simply generate it when performing the test. That option is best used with real unit tests, rather than with doc tests. I would advocate that data added to `xclim-testdata` should be real, not random. 

Also, if we can't find an easy way to have doctest, I say we could just remove that doctest. Either by tagging each line with `# xdoctest: +SKIP` so the example is not run, it only serves as... an example.
OR remove it entirely. Usage of xclim indicators is rather uniform, one should understand from the signature.

Reply:
@aulemahal in ERA5, `"convective_precipitation"`. The var would need to be renamed to `prc`. To make things easier, you can make a request for both `pr` (`"total_precipitation"`) and `prc` so they're in the same file to begin with.

Reply:
We can also remove this example completely, since it's not adding much value at the method here. Would you be OK with that?

Reply:
I am ok with that. We don't need an example for each indicator, this one doesn't seem to be more complex than the others.

Reply:
In that case, I would say just follow @aulemahal's suggestion and add `# xdoctest: +SKIP` to the doctest lines.